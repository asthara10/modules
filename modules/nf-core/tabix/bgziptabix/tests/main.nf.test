nextflow_process {

    name "Test Process TABIX_BGZIPTABIX"
    script "../main.nf"
    config "./nextflow.config"
    process "TABIX_BGZIPTABIX"

    tag "modules"
    tag "modules_nfcore"
    tag "tabix"
    tag "tabix/bgziptabix"

    test("test_tabix_bgziptabix") {


        when {
            process {
                """
                
                input[0] = [ [ id:'test' ], // meta map
              [ file(params.test_data['sarscov2']['illumina']['test_vcf'], checkIfExists: true) ]
            ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
