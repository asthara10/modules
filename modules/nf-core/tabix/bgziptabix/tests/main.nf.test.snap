{
    "versions": {
        "content": [
            [
                "versions.yml:md5,dd2967d0512f0249407f797bb54b33a3"
            ]
        ],
        "timestamp": "2023-11-24T12:44:52.325822"
    }
}